internal/librariangen/generate/generator.go

```diff
@@ -29,6 +29,7 @@ import (
 	"cloud.google.com/go/internal/postprocessor/librarian/librariangen/postprocessor"
 	"cloud.google.com/go/internal/postprocessor/librarian/librariangen/protoc"
 	"cloud.google.com/go/internal/postprocessor/librarian/librariangen/request"
+	"github.com/ghodss/yaml"
```

Comment:
For now could we use the same lib as librarian? gopkg.in/yaml.v3

It is not perfect either but the above library has not had an updated in 4 years.

Reply:
Sure, no problem. I'll switch it.